crates/core/src/trap.rs

```diff
@@ -284,6 +284,13 @@ pub enum TrapCode {
     /// internal bytecode so that fuel is consumed for each executed instruction.
     /// This is useful to deterministically halt or yield a WebAssembly execution.
     OutOfFuel,
+
+    /// This trap is raised when a growth operation was attempted and an
+    /// installed `ResourceLimiter` returned `Err(...)` from the associated
```

Comment:
```suggestion
    /// installed [`ResourceLimiter`](crate::ResourceLimiter) returned `Err(...)` from the associated
```

Reply:
We usually try to make use of intra-doc links in docs to improve UX for users when browsing the generated docs. Actually we even do this for crate-private docs (which is why our CI is checking it) because sometimes using `cargo doc --document-private-items` can be super helpful, too.

Reply:
Done.